XenAdmin/SettingsPanels/LogDestinationEditPage.cs

```diff
@@ -30,168 +30,140 @@
  */
 
 using System;
-using System.Collections.Generic;
-using System.ComponentModel;
 using System.Drawing;
-using System.Data;
-using System.Text;
 using System.Text.RegularExpressions;
 using System.Windows.Forms;
-
-using XenAdmin;
-using XenAdmin.Network;
-using XenAdmin.Core;
 using XenAPI;
-using XenAdmin.Dialogs;
 using XenAdmin.Actions;
+using XenAdmin.Core;
 
 
 namespace XenAdmin.SettingsPanels
 {
     public partial class LogDestinationEditPage : UserControl, IEditPage
     {
-        private Host TheHost;
-
-        private bool _ValidToSave = true;
-        private string _OrigLocation = null;
+        private Host _host;
+        private bool _validToSave;
+        private string _origLocation;
 
         private Regex regex = new Regex(@"^[a-zA-Z0-9]([-a-zA-Z0-9]{0,61}[a-zA-Z0-9])?(\.[a-zA-Z0-9]([-a-zA-Z0-9]{0,61}[a-zA-Z0-9])?)*$");
 
         private readonly ToolTip InvalidParamToolTip;
 
-        public bool ValidToSave
-        {
-            get { return _ValidToSave; }
-        }
-
         public LogDestinationEditPage()
         {
             InitializeComponent();
 
             Text = Messages.LOG_DESTINATION;
 
-            InvalidParamToolTip = new ToolTip();
-            InvalidParamToolTip.IsBalloon = true;
-            InvalidParamToolTip.ToolTipIcon = ToolTipIcon.Warning;
-            InvalidParamToolTip.ToolTipTitle = Messages.INVALID_PARAMETER;
+            InvalidParamToolTip = new ToolTip
+            {
+                IsBalloon = true,
+                ToolTipIcon = ToolTipIcon.Warning,
+                ToolTipTitle = Messages.INVALID_PARAMETER
+            };
         }
 
-        public void SetXenObjects(IXenObject orig, IXenObject clone)
+        private string RemoteServer => ServerTextBox.Text.Trim();
+
+        #region IVerticalTabs implementation
+
+        public Image Image => Properties.Resources.log_destination_16;
+
+        public string SubText
         {
-            TheHost = clone as Host;
-            Repopulate();
+            get
+            {
+                return checkBoxRemote.Checked
+                    ? string.Format(Messages.LOCAL_AND_REMOTE, RemoteServer)

```

Comment:
We might want to show the same text on the general tab (Log destination entry there shows "Local" or the remote server

Reply:
Done; commits can be squashed when merging.